src/Team-Services-Bot.Api/Model/VstsAccount.cs

```diff
@@ -0,0 +1,39 @@
+﻿// ———————————————————————————————
+// <copyright file="VstsAccount.cs">
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+// <summary>
+// Represents a Team Services Account Information.
+// </summary>
+// ———————————————————————————————
+
+namespace Vsar.TSBot
+{
+    using System;
+    using System.Runtime.Serialization;
+
+    /// <summary>
+    /// Represents information about VSTS Account
+    /// </summary>
+    [DataContract]
+    public class VstsAccount
```

Comment:
Vladimir,
Could you please explain why we need to use a separate url, instead of building up the url from account like:
https://{account}.visualstudio.com.

Because the Release Management and package management url differs:
https://{account}.rm.visualstudio.com
https://{account}.pkgs.visualstudio.com

So we have to store these as well then?